systers_portal/templates/base.html

```diff
@@ -57,6 +58,7 @@
               </li>
           </ul>
         </li>
+        <li><a href="{% url 'list_meetup_location' %}" target="blank_">Meetup Locations</a></li>
```

Comment:
It's not an external link, so probably should remove the `target="blank_"`. Let's keep it all portal.systers.org links in the same tab.